CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+## Unreleased
+
+
+## [v0.1.4](https://github.com/omisego/elixir-omg/releases/tag/v0.1.4) 
```

Comment:
I've done a quick attempt to list the changes, and I ended up with a slightly different list:

* bump version to v0.1.4
* Merge pull request #583 from omisego/silence_socket_connect_logs 7dab59
* fix: silence channel join logs ba1331
* bump version to v0.1.3
* Merge pull request #514 from omisego/update_launcher_logic_watcher 5ee21b
* Merge pull request #512 from omisego/increase_restarts 7c1e99
* Merge pull request #498 from omisego/safeguard_against_lengthy_get_eth_events fe2e2d
* bump version to v0.1.2
* Merge pull request #490 from omisego/OMG-218-appsignal_phoenix_instrumentation ecdb1a
* Merge pull request #487 from omisego/OMG-218-appsignal_phoenix_instrumentation 081f7c
* Merge pull request #477 from omisego/OMG-218-appsignal_phoenix_instrumentation 13c10c
* Merge pull request #470 from omisego/OMG-417-status_get_breaks_if_geth_is_syncing a36692
* Merge pull request #463 from omisego/OMG-437_sentry_crashes ba4b7d
* fix: #474 from omisego/OMG-414_Idempotent_writes_deposits_exits_to_WatcherDB d82a75
* Merge pull request #482 from omisego/ignore_processing_exit_finalized_watcher_events 0b1447
* Merge pull request #481 from omisego/v0.1_cirlc 00e629
* Merge pull request #469 from omisego/update_ci_config c45729
* bump version to v0.1.1

e.g. I have many more between 0.1.1 and 0.1.2 (the above is an excerpt from my commit log of `v0.1`)

(I stopped on 0.1.1)

Reply:
I'm not opinionated on the naming, happy to just scrape the PR titles as long as we keep the changelog formats consistent within all the repos :)